schemas/constants.yml

```diff
@@ -1,2 +1,2 @@
 access-token-pattern:     "^[a-zA-Z0-9_-]{22,66}$"
-clientId: "^[A-Za-z0-9@/:._-]+$"
+clientId: "^[A-Za-z0-9%@/:._-]+$"
```

Comment:
Uhoh, this is defined at `^[A-Za-z0-9@/:.+|_-]+$` in the auth service.  Interesting that it includes `|` there -- that was a [recent addition](https://github.com/taskcluster/taskcluster-auth/commit/57e7da04fe0a49cba8347fd44e4927f36ac4c05f) that didn't get reflected here.  Perhaps we can also add %?  @jonasfj?